index.bs

```diff
@@ -1831,19 +1853,19 @@ a numbered step. If outdented, it (today) is rendered either as a bullet in the
 
                 1.  If <code>|credentialCreationData|.[=attestationConveyancePreferenceOption=]</code>'s value is
                     <dl class="switch">
-                        :   "none"
+                        :   {{AttestationConveyancePreference/none}}
```

Comment:
I was thinking these names are references to predefined values. This also seems to be the (perhaps accidental) current convention in the spec: this is currently the only switch statement with double quotes for the case values and the only one using reference-less literals instead of references.

Reply:
again, iiuc, these are actually literal DOMString values, are they not?

Reply:
It is, but we're not currently using quotes around references to enum values anywhere else, see for example:

- https://github.com/w3c/webauthn/blob/38f9f7257914968292ca421b86a2db1670131754/index.bs#L1515
- https://github.com/w3c/webauthn/blob/38f9f7257914968292ca421b86a2db1670131754/index.bs#L1665
- https://github.com/w3c/webauthn/blob/38f9f7257914968292ca421b86a2db1670131754/index.bs#L2763

We could introduce quotes here, but then it seems to me like we should do it everywhere. I would be okay with doing that, but I don't think it seems necessary.

Reply:
sigh.  :shrug:

Reply:
No worries, thanks for your patience :slightly_smiling_face: